packages/kos-mobile/build_source.sh

```
@@ -2,9 +2,26 @@
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  OS="darwin-x86_64"
+  OS_ARCH="darwin64-arm64-cc"
+  OS_TOOLCHAIN="darwin-x86_64"
+  IS_MACOS=true
+  LIB_EXTENSION="dylib"
+  JNI_PLATFORM="darwin-aarch64"
+elif [[ "$OSTYPE" == "linux"* ]]; then
+  OS="linux-x86_64"
+  OS_ARCH="linux-x86_64"
+  OS_TOOLCHAIN="linux-x86_64"
```

Shellcheck (0.10.0)

[warning] OS_TOOLCHAIN appears unused. Verify use (or export if used externally).

(SC2034)